AI/app/data/util/plan_converter.py

```
@@ -0,0 +1,189 @@
+                season = "여름"
+            elif month in [9, 10, 11]:
+                season = "가을"
+            elif month in [12, 1, 2]:
+                season = "겨울"
+
+        # 식당 포함 여부
+        has_restaurants = any(attraction.get('categoryId') == 39 for attraction in attractions)
+
+        # 완성도 점수 (제목, 내용, 관광지 유무)
+        completeness_score = 0
+        if base_row.title: completeness_score += 0.3
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)